packages/frontend/package.json

```diff
@@ -35,8 +35,7 @@
         "config-react-env": "./scripts/config-react-env.js",
         "start:staging": "STAGING=true craco start",
         "start": "craco start",
-        "test:e2e": "yarn prepare-fathom && playwright test",
```

Comment:
We can omit the `yarn` before `playwright test`. so just have:

`yarn prepare-fathom && playwright test`

Reply:
I will edit it now but I am getting error `command not found: playwright test` if I ommit yarn

Reply:
Running it on my machine it works fine without the `yarn`, anyway let's leave it if it causes issues.